apps/admin-web/src/app/components/RouteItem.tsx

```diff
@@ -1,119 +1,189 @@
+    let backgroundColor;
+    let textColor;
+    switch (route.method) {
+      case 'GET':
+        backgroundColor = blue[50];
+        textColor = blue[800];
+        break;
+      case 'POST':
+        backgroundColor = green[50];
+        textColor = green[800];
+        break;
+      case 'PUT':
+        backgroundColor = purple[50];
+        textColor = purple[800];
+        break;
+      case 'DELETE':
+        backgroundColor = red[50];
+        textColor = red[800];
+        break;
+      case 'PATCH':
+        backgroundColor = orange[50];
+        textColor = orange[800];
+        break;
+      default:
+        break;
+    }
+
+    return { backgroundColor, textColor };
+  };
+
+  const _renderRouteTitle = () => {
+    return (
+      <Typography variant="body1" sx={{ p: 2 }} style={{ color: 'black' }}>
+        {route.path}
+      </Typography>
+    );
+  };
+
+  const _renderInteractiveButtons = () => {
+    return (
+      <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+        {/* I think this is too much, I thought it was a button and was confused when it didn't work, accordion is enough */}
+        {/* {selectedItem === route.id && (
+          <Visibility
+            color="primary"
+            sx={{ alignSelf: 'center', mr: 1 }}
+          ></Visibility>
+        )} */}
+        <Button
+          variant="contained"
+          size="small"
+          sx={{
+            mr: 2,
+            backgroundColor: getColors().textColor,
+            maxHeight: 30,
+            alignSelf: 'center',
+          }}
+          onClick={() => {
+            if (route.method?.toUpperCase() === 'GET') {
+              openInNewTab(route.path.toString());
+            } else {
+              // If not a POST, currently assumes response is always JSON, fetch via API then open
+              fetch(route.path.toString(), {
+                method: route.method,
+                headers: {
+                  Accept: 'application/json',
+                  'Content-Type': 'application/json',
+                },
+                body: route.method === 'GET' ? undefined : '{}',
+              })
+                .then((r) => r.json())
+                .then(openJsonInNewTab);
+            }
+          }}
+        >
+          <Bookmark />
+        </Button>
+        <Container
+          sx={{
+            backgroundColor: getColors().textColor,
+            pt: '15px',
+            width: '100px',
+          }}
+        >
+          <Typography sx={{ color: 'white', textAlign: 'center' }}>
+            {route.method}
+          </Typography>
+        </Container>
+      </Box>
+    );
+  };
+
   return (
-    <Paper style={{ backgroundColor }}>
-      <Grid
-        container
-        direction="row"
-        justifyContent="space-between"
-        alignItems="center"
+    <Paper
+      style={{
+        backgroundColor: getColors().backgroundColor,
+        overflow: 'hidden',
+      }}
+      onClick={() => setSelectedItem(route.id)}
```

Comment:
Quality of life thing to close the route, if you click on it again. 

```suggestion
      onClick={() => route.id === selectedItem ? setSelectedItem("") : setSelectedItem(route.id)}
```